code/modules/spacepods/actions.dm

```diff
@@ -0,0 +1,104 @@
+
+
+/obj/spacepod/proc/GrantActions(mob/living/user)
+	exit_action.Grant(user, src)
+	lock_action.Grant(user, src)
+	door_action.Grant(user, src)
+	fire_action.Grant(user, src)
+	unload_action.Grant(user, src)
+	light_action.Grant(user, src)
+	seat_action.Grant(user, src)
+	tank_action.Grant(user, src)
+
+
+/obj/spacepod/proc/RemoveActions(mob/living/user)
+	exit_action.Remove(user)
+	lock_action.Remove(user)
+	door_action.Remove(user)
+	fire_action.Remove(user)
+	unload_action.Remove(user)
+	light_action.Remove(user)
+	seat_action.Remove(user)
+	tank_action.Remove(user)
+
+/datum/action/innate/spacepod
+	var/obj/spacepod/S
+	check_flags = AB_CHECK_RESTRAINED | AB_CHECK_STUN | AB_CHECK_CONSCIOUS
+
+/datum/action/innate/spacepod/Grant(mob/living/L, obj/spacepod/M)
+	if(M)
```

Comment:
In spacepod.dm, if the pod is destroyed or the pilot exits (or is jerked out), they automatically have the actions remove